content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -1,5 +1,58 @@
 ---
 title: "Workload Resources"
 weight: 20
+simple_list: true
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+Ultimately, your applications run as containers inside
+{{< glossary_tooltip term_id="Pod" text="Pods" >}}; however, managing individual
+Pods would be a lot of effort. For example, if a `Pod` fails, you probably want to
+run a new Pod to replace it. Kubernetes can do that for you.
+
+You use Kubernetes API to create a workload
+{{< glossary_tooltip text="object" term_id="object" >}} that represent a higher level
+abstraction than a `Pod`, and then the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manages
+Pod objects on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) (and, indirectly, [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)),
+the most common way to run an application on your cluster.
+A Deployment is a good fit for managing a stateless application workload on your cluster:
+_stateless_ means that any {{< glossary_tooltip term_id="Pod" >}} in the Deployment is interchangeable,
+and can be replaced if needed.
+(Deployments are a replacement for the legacy
+{{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}} API).
+
+A [`StatefulSet`](/docs/concepts/workloads/controllers/statefulset/) lets you
+manage one or more Pods – all running the same application code – that do track state somehow.
+For example, if your workload records data persistently, you can run a StatefulSet that
```

Comment:
The key difference between *stateless* and *stateful* could be this: whether the workload records data persistently and collaboratively. It is okay for a Deployment Pod to use PV volumes, that doesn't make it stateful.

Reply:
I'll have a think if I can come up with a concise explanation that really captures the nuances here.

Reply:
How's the revised text?